pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -909,6 +887,32 @@ protected synchronized boolean trySendMessagesToConsumers(ReadType readType, Lis
         return true;
     }
 
+
+    @VisibleForTesting
+    int getMaxEntriesInThisBatch(int remainingMessages, Consumer c, int avgBatchSizePerMsg) {
```

Comment:
to unit test just this method, one option is to make this method a static method so that the unit test doesn't become an integration test. In Pulsar, most "unit tests" are actually integration tests and here the intention seems to be to unit test this single method.